projects/plugins/boost/app/lib/Features_Status.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Class to check the status of Boost features.
+ *
+ * @package automattic/jetpack-boost
+ */
+
+namespace Automattic\Jetpack_Boost\Lib;
+
+use Automattic\Jetpack_Boost\Lib\Critical_CSS\Critical_CSS_State;
+
+/**
+ * Class that handles checking the status of Jetpack Boost features.
+ */
+class Features_Status {
+
+	const GOOD = 'good';
+	const BAD  = 'bad';
```

Comment:
If the only purpose of `good` and `bad` are to identify if an error exists or not, the flag could be just a boolean. If we think in terms of `has_cloud_css_problem` instead of `get_cloud_css_state_status`, a boolean would be sufficient. I think that would also simplify the class and make it more intuitive.

Reply:
Fixed with 08499227213e2db9d865e1de077ac2552230c1a5.

I guess I was thinking around the idea that a feature has three states - good, bad and inactive. I then decided to combine inactive and good into one, but I didn't re-think the whole logic.

Also - great suggestion with the method name! That made the code far easier to read.